MSBuildTasks.Project.nuspec

```diff
@@ -9,15 +9,12 @@
     <projectUrl>https://github.com/loresoft/msbuildtasks/</projectUrl>
     <licenseUrl>http://opensource.org/licenses/bsd-license.php</licenseUrl>
     <tags>MSBuild</tags>
+    <developmentDependency>true</developmentDependency>
   </metadata>
   <files>
-    <file src="Build\MSBuild.Community.Tasks.dll" target="tools\MSBuild.Community.Tasks.dll" />
-    <file src="Build\MSBuild.Community.Tasks.xml" target="tools\MSBuild.Community.Tasks.xml" />
-    <file src="Build\MSBuild.Community.Tasks.Targets" target="tools\MSBuild.Community.Tasks.Targets" />
+    <file src="Build\MSBuild.Community.Tasks.dll" target="build" />
+    <file src="Build\MSBuild.Community.Tasks.xml" target="build" />
+    <file src="Build\MSBuild.Community.Tasks.Targets" target="build" />
```

Comment:
These files are used as part of the build system, not as part of the project being built.  The build folder is the correct place for such files.